src/responder/secrets/local.c

```diff
@@ -447,6 +468,9 @@ static int local_db_create(TALLOC_CTX *mem_ctx,
     ret = local_db_check_containers(msg, lctx, msg->dn);
     if (ret != EOK) goto done;
 
+    ret = local_db_check_containers_nested_level(lctx, msg->dn);
+    if (ret != EOK) goto done;
```

Comment:
I can rebase my patches on top of yours, that wouldn't be a problem. Let me know what's your preference.